recipes/kibana/Dockerfile

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
Probably, I was actually talking with @phildav  on how we could properly package the final version with pre-built images for oncrawl-elk of nginx, kibana with its config, etc..

I'll add the rename of this image to the list :)